tests/safe_command/fuzzdb/command-injection-template.txt

```diff
@@ -0,0 +1,45 @@
+{cmd}
```

Comment:
https://github.com/fuzzdb-project/fuzzdb/blob/master/attack/os-cmd-execution/command-injection-template.txt

Note I removed the SSI and URL-encoded payloads since they are not what we are testing here.